docs/cloud-messaging/client.md

```diff
@@ -144,8 +144,20 @@ To retrieve the current registration token for an app instance, call
 ask the user for notification permissions. Otherwise, it returns a token or
 rejects the future due to an error.
 
+Warning: From iOS SDK 10.4.0 and higher, it is a requirement that the APNS token
```

Comment:
hey @kevinthecheung - documentation update that could do with a review, please 😄